cicd/pipelines/using-pipelines-as-code.adoc

```diff
@@ -9,15 +9,15 @@ toc::[]
 // :FeatureName: Pipelines as Code
 
 [role="_abstract"]
-With {pac}, cluster administrators and users with the required privileges can define pipeline templates as part of source code Git repositories. When triggered by a source code push or a pull request for the configured Git repository, the feature runs the pipeline and reports the status.   
+With {pac}, cluster administrators and users with the required privileges can define pipeline templates as part of source code Git repositories. When triggered by a source code push or a pull request for the configured Git repository, the feature runs the pipeline and reports the status.
```

Comment:
are you calling  a feature? i guess it's more like a component or controller, up to you to choose the better name